### DISCUSSION TITLE
What is domain fronting, and how to set it up in Hiddify panel? دامین فرانتینگ چیه و چه طوری تو پنل هیدیفای تنظیمش کنیم؟

### DISCUSSION THREAD

**Discussion by TheTraxex** [hiddify/Hiddify-Manager#119] [Q&A سوالات رایج] [answered]
دامین فرانتینگ چیه و چه طوری تو پنل هیدیفای تنظیمش کنیم؟

**Comment by hiddify-com** [comment, marked as answer]
<div dir="rtl" markdown="1">
ابتدا [این لینک](https://paskoocheh.com/blog/posts/2018-04-20-how-domain-fronting-works.html?nocookies)  را مشاهده کنید تا با مفهوم دامین فرانتینگ آشنا بشید

پس از آن کافی است که در سایت cloudflare.com ثبت نام کنید و از سرویس دامنه (DNS) آنها استفاده کنید.

[این فیلم که برای کلود فلر هست را مشاهده کنید](https://www.youtube.com/watch?v=Sgqnznm6SEY)

فقط در قسمت آخر 



حالا آدرس یک سایتی که از این سرویس ها استفاده میکنند و فیلتر نیستند را در تنظیمات سیستم تایپ کنید مطابق شکل زیر
![image](https://user-images.githubusercontent.com/114227601/212904324-0a485db0-4b78-4d2e-b363-df0f42483013.png)




یا siencedirect.com اگه از کلودفلر استفاده میکنید ([لیست سایت های موجود در کلودفلر](https://trends.builtwith.com/cdn/Cloudflare/Iran)) یا [لیست سایت های کلود فلر در ایران](https://index.woorank.com/en/reviews?countries=IR&technologies=cloudflare)

[ابزار DNS Lookup](https://dns-lookup.com) برای بررسی اینکه سایتها در کلود فلر  هستند
</div>
